src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -156,6 +156,10 @@ public abstract partial class JsonNamingPolicy
     {
         protected JsonNamingPolicy() { }
         public static System.Text.Json.JsonNamingPolicy CamelCase { get { throw null; } }
+        public static System.Text.Json.JsonNamingPolicy SnakeLowerCase { get { throw null; } }
```

Comment:
This doesn't match the APIs as approved in https://github.com/dotnet/runtime/issues/782#issue-536684456. A few remarks

* Do we need to add kebab-case support OOTB in STJ?
* How common are uppercase variants and do we need to ship them with STJ?
* My understanding is that snake_case is always lowercase, whereas uppercase is called SCREAMING_SNAKE_CASE. Naming convention needs to be determined in API review.

If you feel that the additional functionality is needed, we need to bring an updated proposal to API review.

Reply:
I would like to ship all these policies because they exist in other serializers. They might be not all widely used, but they're still used. It costs nothing as you can see to support them.

Speaking about names, there are no rules about that as well as conversion rules. Since both snake and kebab can be upper case as well as lower case, it makes more sense to tell which separator is used and case, and keep them symmetrical. That's more intuitive.

Reply:
> It costs nothing as you can see to support them.

New APIs cannot be added without a review and approval that precisely defines their shape, and the bar for approving them is much higher than "it costs nothing as you can see to support them".

Reply:
Yeah, I know that. The problem is that .NET isn't a single platform in the world, but one of them, and it should be friendly for interop. Otherwise, it will loose positions. I will update the proposal, just sat to the computer again.